int32[] dxl_ids
int32[] registers
---
int32[] values
float32[] fvalues
int32[] results
byte[] error_bits
